documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -508,11 +498,11 @@ Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rul
 
 **Description**: Verifies whether operation supports "application/json" as consumes or produces section.
 
-**Why the rule is important**: N/A.
+**Why the rule is important**: Per the ARM SDK guidelines, the AutoRest is used to generated the language SDKs. All the supported language generator currently handles "application/json" but some SDKs may support other content-types. This rule is in place to warn the swagger writter that not all the SDKs would work with other content-types.
 
 **How to fix the violation**: N/A.
```

Comment:
Should we say something like: if the service supports other content types, don't fix the warning, look into suppression mechanism, if the other content types are not supported, please remove them from the spec?
I think all rules have to include "how to fix the violation", if not it's confusing for the consumer.